.github/workflows/builds_and_tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - name: Install Rust
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.54.0
+          toolchain: 1.59.0
```

Comment:
Is it no problem? 
I heard the appropriate rustc version is 1.57.0

Reply:
thx! I changed to 1.57.0

Reply:
Now, the toolchain version of Dockerfiles is 1.60.0. If there are no reasons we cannot unify, we should do.